_includes/head.html

```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
```

Comment:
:+1: Aligning with `og:description` here makes sense, assuming that value isn't more than [~155 characters](https://moz.com/learn/seo/meta-description).